pandas/tests/tools/test_to_datetime.py

```diff
@@ -2263,73 +2206,23 @@ def test_to_datetime_infer_datetime_format_consistent_format(
-        no_infer = to_datetime(
-            s_as_dt_strings, infer_datetime_format=False, cache=cache
-        )
-        yes_infer = to_datetime(
-            s_as_dt_strings, infer_datetime_format=True, cache=cache
-        )
+        no_infer = to_datetime(s_as_dt_strings, cache=cache)
+        yes_infer = to_datetime(s_as_dt_strings, cache=cache)
 
         # Whether the format is explicitly passed, it is inferred, or
         # it is not inferred, the results should all be the same
         tm.assert_series_equal(with_format, no_infer)
         tm.assert_series_equal(no_infer, yes_infer)
 
     @pytest.mark.parametrize(
-        "data",
-        [
-            ["01/01/2011 00:00:00", "01-02-2011 00:00:00", "2011-01-03T00:00:00"],
-            ["Jan/01/2011", "Feb/01/2011", "Mar/01/2011"],
-        ],
+        "tz_name, offset, warning",
+        [("UTC", 0, None), ("UTC-3", 180, UserWarning), ("UTC+3", -180, UserWarning)],
     )
-    def test_to_datetime_infer_datetime_format_inconsistent_format(self, cache, data):
-        ser = Series(np.array(data))
-
-        # When the format is inconsistent, infer_datetime_format should just
-        # fallback to the default parsing
-        tm.assert_series_equal(
-            to_datetime(ser, infer_datetime_format=False, cache=cache),
-            to_datetime(ser, infer_datetime_format=True, cache=cache),
-        )
-
-    def test_to_datetime_infer_datetime_format_series_with_nans(self, cache):
-        ser = Series(
-            np.array(
-                ["01/01/2011 00:00:00", np.nan, "01/03/2011 00:00:00", np.nan],
-                dtype=object,
-            )
-        )
-        tm.assert_series_equal(
-            to_datetime(ser, infer_datetime_format=False, cache=cache),
-            to_datetime(ser, infer_datetime_format=True, cache=cache),
-        )
-
-    def test_to_datetime_infer_datetime_format_series_start_with_nans(self, cache):
-        ser = Series(
-            np.array(
-                [
-                    np.nan,
-                    np.nan,
-                    "01/01/2011 00:00:00",
-                    "01/02/2011 00:00:00",
-                    "01/03/2011 00:00:00",
-                ],
-                dtype=object,
-            )
-        )
-
-        tm.assert_series_equal(
-            to_datetime(ser, infer_datetime_format=False, cache=cache),
-            to_datetime(ser, infer_datetime_format=True, cache=cache),
-        )
-
-    @pytest.mark.parametrize(
-        "tz_name, offset", [("UTC", 0), ("UTC-3", 180), ("UTC+3", -180)]
-    )
-    def test_infer_datetime_format_tz_name(self, tz_name, offset):
+    def test_infer_datetime_format_tz_name(self, tz_name, offset, warning):
         # GH 33133
         ser = Series([f"2019-02-02 08:07:13 {tz_name}"])
-        result = to_datetime(ser, infer_datetime_format=True)
+        with tm.assert_produces_warning(warning, match="Could not infer format"):
+            result = to_datetime(ser)
```

Comment:
Not clear why we can't infer the format here.  Is it becauase the month and day are both 2?  If so, maybe change the string to use `f"2019-02-24 08:07:13 {tz_name}"` since the point of the test is to test timezone names

Reply:
It's because the timezone is `UTC-3`. it'd work if it 
```
In [1]: guess_datetime_format('2019-02-02 08:07:13 UTC-3')
```
It'd work if it was
```
In [2]: guess_datetime_format('2019-02-02 08:07:13 -03:00')
Out[2]: '%Y-%m-%d %H:%M:%S %z'
```